cookbook/case_studies/ml_training/house_price_prediction/house_price_predictor.py

```diff
@@ -44,12 +64,14 @@
     "GARAGE_SPACES",
 ]
 MAX_YEAR = 2021
+# Now, we divide the data into train, validation, and test datasets in specific ratio.
```

Comment:
Is this a comment or should this be included above the code?

Reply:
its a comment..